js/apps/admin-ui/maven-resources/theme/keycloak.v2/admin/messages/messages_en.properties

```diff
@@ -3605,3 +3605,5 @@ credentialLifetime=Credential Lifetime (seconds)
 credentialLifetimeHelp=The lifetime of the credential in seconds. After this time, the credential will expire and become invalid.
 supportedFormats=Supported Formats
 supportedFormatsHelp=The format of the verifiable credential. Currently supported formats: SD-JWT VC (dc+sd-jwt), JWT VC (jwt_vc_json).
+hideFromOpenIdProviderMetadata=Hide from OpenID Provider Metadata
+hideFromOpenIdProviderMetadataHelp=If on, this client scope will be hidden from OpenID Provider Metadata.
```

Comment:
There is an existing option in the same dialog with the "Include in token scope", and this new option is inverting it by stating "Hide from..."

Please rework this new option to also be name "Include in ...". When doing this, this needs to be inverted to be true by default both in the back end and in the front end.